src/tests/test_namespace_pkgs.py

```diff
@@ -0,0 +1,115 @@
+import os
+import pathlib
+import unittest
```

Comment:
Could've used `pytest`, but this works fine and keeps us at zero `pip`  dependencies 👍.